website/views/user.py

```
@@ -979,44 +1134,1123 @@ def badge_user_list(request, badge_id):
+    repo_full_name = pr_data["base"]["repo"]["full_name"]
+    repo_name = repo_full_name.split("/")[-1]
+    repo = Repo.objects.filter(name__iexact=repo_name).first()
+
+    if not repo:
+        logger.warning("Repo not tracked: %s", repo_full_name)
+        return
+
+    try:
+        usd_amount_dec = Decimal(str(usd_amount))
+    except (InvalidOperation, TypeError, ValueError):
+        logger.error("Invalid usd_amount passed to process_bounty_payment: %r", usd_amount)
```

Ruff (0.14.5)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)